packages/test-helpers/src/setup.js

```diff
@@ -4,13 +4,16 @@ import log from "../../common/logger";
 import checkIfWebStarted from "./didWebStart";
 import Snapper from "../../snapshots/snapper";
 import { disconnectClient } from "./withComponent";
+import fructoseClient from "../../client";
 
 const mobileHooks = () => {
   let server;
 
   const setup = async () => {
     server = new FructoseServer(7811);
 
+    global.fructoseClient = fructoseClient(7811);
```

Comment:
can we return the client as the resolution of the promise rather than setting it as a global.